generate_librimix_sd.sh

```diff
@@ -79,5 +78,5 @@ for n_src in 2; do
     --n_src $n_src \
     --freqs 16k \
     --modes max \
-    --types mix_clean mix_both
+    --types mix_both
```

Comment:
Can @ftshijt please help me proof-read this?
Thanks!

Reply:
LGTM! Thanks